### ISSUE TITLE
segfault

### ISSUE THREAD

**Issue by kim-em** [leanprover/lean3#1545]
````
git clone https://github.com/semorrison/lean-category-theory
cd lean-category-theory
git reset --hard ebcd6ec24ecea688e2b2b95c30ec38ac72ec0c92
lean --make natural_transformation.lean
````

**Comment by kim-em** [comment]
It seems to be in the definition of [IdentityNaturalTransformation](https://github.com/semorrison/lean-category-theory/blob/master/natural_transformation.lean#L40).

**Comment by kim-em** [comment]
I'll close this. The segfault itself was solved by bff4203f1716d143d6647c3a2f1bf5aec2ea2f42, and the 'real' problem is now described in #1552.
